app.rb

```diff
@@ -0,0 +1,28 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/peep.rb'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do
+    @peeps = Peep.all
+    erb :'/index'
+  end
+
+  post '/add_peep' do
```

Comment:
Correct me if I'm wrong, it might be worth looking into REST (click [here](https://github.com/makersacademy/course/blob/main/pills/rest.md) to look at the pill). It talks about url naming conventions, let me know what you think.